exercism/hamming/hamming.src.js

```diff
@@ -0,0 +1,17 @@
+var Hamming = function() {};
+
+Hamming.prototype.compute =  function (a, b) {
+  if (a.length === b.length) {
+  var diff = 0;
+    for (var i = 0; i <= a.length; i++) {
+	  if (a[i] != b[i]) {
+	    diff++;            
+      };
+  	};
```

Comment:
line 9 is closing the ```if``` statement on line 7 and line 10 is closing the ```for``` loop on line 6 so you want the indentation of your closing brackets to line up